### ISSUE TITLE
Option to customize error messages

### ISSUE THREAD

**Issue by zbicin** [sagrawal14/bootstrap-angular-validation#10]
I though it'd be nice if the library allowed to globally alter the error messages being displayed. E.g.

``` javascript
var app = angular.module("bavApp", ["bootstrap.angular.validation"]);

app.config(['bsValidationConfigProvider', function(bsValidationConfigProvider) {
  bsValidationConfigProvider.global.messages.required = "Come on, I can't work without it";
}]);
```

**Comment by sagrawal31** [comment]
Thanks for opening this issue. I was planning to work on this next 😄

**Comment by zbicin** [comment]
Great! Support for custom errors would be also nice. I assume you lookup the $error array of a given form field, so it shouldn't be a big deal. :)

**Comment by sagrawal31** [comment]
Yes, I'll modify that in that way so that other than the default error messages, other error messages can be customized globally. The error messages resolution works based on the `$error` array.

**Comment by sagrawal31** [comment]
@zbicin Released new version v1.0.1 with this change of customizing global error messages. You can customize messages like you mentioned (I guess that is the cleanest way of doing this and jQuery validation example also do the same):

``` javascript
var app = angular.module("bavApp", ["bootstrap.angular.validation"]);

app.config(['bsValidationConfigProvider', function(bsValidationConfigProvider) {
  bsValidationConfigProvider.global.messages.required = "Come on, I can't work without it";
}]);
```

Will post example in the site soon.

**Comment by zbicin** [comment]
Cool, that's great news! Thanks.
